public_src/components/modal/modal.component.ts

```diff
@@ -318,9 +331,44 @@ export class ModalComponent {
       val = 'false';
       popupContent.innerHTML = '<i class="fa fa-exclamation-triangle" aria-hidden="true"></i>';
     }
-    this.errorObject.corrected = val;
-    this.storageSrv.refErrorsO[this.storageSrv.currentIndex].corrected = val;
+    this.refErrorObject.corrected = val;
+    this.storageSrv.refErrorsObj[this.storageSrv.currentIndex].corrected = val;
     let popupArr: any      = this.mapSrv.popUpLayerGroup.getLayers();
     popupArr[0].setContent(popupContent);
   }
+
+  /***
+   * Determines if new refs were added
+   * @returns {boolean}
+   */
+  viewAddedRefs(): boolean {
+    return this.newAddedRefs.length !== 0 ||
+      this.addedMissingSuggestionsRefs.length !== 0 ||
+      this.addedFromNearbySuggestionsRefs.length !== 0;
+  }
+
+  /***
+   * Returns tool tip text
+   * @param {string} name
+   * @returns {string}
+   */
+  getTooltipText(name: string): string {
```

Comment:
interesting. If we would have more long texts (which I tried to avoid now because tooltips are usually not accessible on mobile phones) around the project, I might be better to create a separate (`tooltip.utils.ts`?) project just to return different strings for tooltips...